include/ccf/pal/attestation_sev_snp.h

```diff
@@ -75,13 +75,28 @@ QPHfbkH0CyPfhl1jWhJFZasCAwEAAQ==
       ecdsa_p384_sha384 = 1
     };
 
+#pragma pack(push, 1)
+    // Table 8
+    struct GuestPolicy
+    {
+      uint8_t abi_minor;
+      uint8_t abi_major;
+      uint8_t smt : 1;
+      uint8_t reserved : 1;
+      uint8_t migrate_ma : 1;
+      uint8_t debug : 1;
+      uint8_t single_socket : 1;
+      uint64_t reserved2 : 43;
```

Comment:
```suggestion
      uint8_t reserved2[344];
```

For consistency with the other structs.

Reply:
Are these the same thing? My understanding is the current is a bit field which is 43 bits long whereas the suggestion is an array of 64 bit values making it 2752 bits long?

Reply:
Ah yes, you're right. I didn't read this right. The equivalent would be `uint8_t reserved2[43]`.

Reply:
Isn't that still an array of 43 uint8_t's (344 bits) as opposed to a uint64_t restricted to 43 bits?

Reply:
Forget what I said!